packages/web/src/app/[domain]/browse/[...path]/components/pureCodePreviewPanel.tsx

```
@@ -0,0 +1,220 @@
+
+    return (
+        <ScrollArea className="h-full overflow-auto flex-1">
+            <CodeMirror
+                className="relative"
+                ref={setEditorRef}
+                value={source}
+                extensions={extensions}
+                readOnly={true}
+                theme={theme}
+            >
+                {editorRef && editorRef.view && currentSelection && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)